### DISCUSSION TITLE
view prom_info.metric encounter error

### DISCUSSION THREAD

**Discussion by carlwang87** [timescale/promscale#1466] [Feedback on Features] [answered]
According to the doc, Querying the metric view returns all metrics collected by Prometheus:
`SELECT *FROM prom_info.metric;`

Error as below:
`
postgres=# select * from prom_info.metric;
ERROR:  column reference "compressed_toast_size" is ambiguous
LINE 5: ... (COALESCE(sum(ch.toast_bytes), 0) + COALESCE(sum(ch.compres...
`

The promscale verison 0.11.0.

**Comment by JamesGuthrie** [comment, marked as answer]
Which version of the promscale extension are you using? You can find this out with `SELECT extversion FROM pg_extension WHERE extname = 'promscale';`.

**Reply by carlwang87** [reply]
extversion: 0.5.0

**Reply by carlwang87** [reply]
I upgrade promscale from 0.11.0 to 0.12.0, this issue still exists.  extversion is still 0.5.0


**helm chart:**
apiVersion: v2
appVersion: 0.12.0
description: Promscale Connector deployment
home: https://github.com/timescale/promscale
name: promscale
version: 0.12.0

**Reply by JamesGuthrie** [reply]
We fixed this issue in version 0.5.1 of the extension.

**Reply by carlwang87** [reply]
How apply 0.5.1 version of extension in the promscale helm chart install? The latest promscale helm chart version 0.12.0 with default 0.5.0.

**Reply by JamesGuthrie** [reply]
How did you install TimescaleDB? If you're using Helm for that too, the promscale extension is packaged with TimescaleDB, so you will need to update to the most recent TimescaleDB.

**Reply by carlwang87** [reply]
It was fixed by upgrading timescaleDB, thank you very much.
